templates/PatchUtilFile.stencil

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+// TODO: Remove this when these objects are in AzureCore
```

Comment:
Do we need PatchUtilFile.swift ? I think we only need PatchUtilFile.stencil for generating PatchUtilFile.swift in generated code.

Reply:
Hmmm... good point. I think I had it in there so I could test it :)

Reply:
originally it was in there because I envisioned creating the patch object in the generator and only outputting the result to the view model. Then we wouldn't need to copy this file to the generated code. But copying would result in fewer changes when AzureCore supports PATCH, so I went that route. I've removed this file.

Reply:
And by "this file" I mean PatchUtil.swift in the autorest.swift repo :)